packages/app/README.md

```diff
@@ -302,7 +303,8 @@ The sources of this package are contained in the Charcoal monorepo. We welcome c
 
 [climate]:               https://packagist.org/packages/league/climate
 [fastroute]:             https://packagist.org/packages/nikic/fast-route
-[flysystem]:             https://packagist.org/packages/league/flysystem
+[event]:                 https://packagist.org/packages/league/event
+[flysystem]:              https://packagist.org/packages/league/flysystem
```

Comment:
I think the reference links should be prefixed with the vendor name to be future-proof.

```md
[league/climate]:        https://packagist.org/packages/league/climate
[league/fastroute]:      https://packagist.org/packages/nikic/fast-route
[league/flysystem]:      https://packagist.org/packages/league/flysystem
[league/event]:          https://packagist.org/packages/league/event
```